### PR TITLE
A Halloween page with a ghostly number guessing game.

### PR DESCRIPTION
**Pull request by MikeStrider**
A Halloween page with a ghostly number guessing game.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +0,0 @@
-.cache
```
Comment (ericjiang97) [inline .gitignore:1]: Why is the gitignore files deleted?

**Comment by ericjiang97** [comment]
Also please add your site to the main page!!!

File: .gitignore (hunk 1 above)
Comment (MikeStrider) [reply .gitignore:1]: I didn't think u needed my git ignore file.  Uploaded it by mistake.

**Comment by MikeStrider** [comment]
ahh yes .. sorry .. working on it now

File: .gitignore (hunk 1 above)
Comment (ericjiang97) [reply .gitignore:1]: Thats the gitignore file that we use, ummm didn't think you uploaded it.

Comment (MikeStrider) [reply .gitignore:1]: i deleted the request to merge my gitignore with your gitignore

**Comment by MikeStrider** [comment]
Ready to go.  Please check out my page and play the short game.

**Review by ericjiang97 — APPROVED** [review]

**Comment by josh-** [comment]
Love the game @MikeStrider, great idea! Thanks for your Pull Request 💯 

Screen Shot 2019-10-18 at 5 22 55 pm

PS I won! 🥳

**Pull request merged**